Project.toml

```
@@ -10,7 +10,7 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 julia = "1.6"
-StaticArraysCore = "1"
+StaticArraysCore = "1.1"
```

```suggestion
StaticArraysCore = "~1.1"
```

https://pkgdocs.julialang.org/v1/compatibility/#Tilde-specifiers